src/backend/central/central_routes.py

```diff
@@ -18,24 +18,23 @@
 
 from fastapi import APIRouter, Depends
 from sqlalchemy.orm import Session
+from sqlalchemy.sql import text
+from sqlalchemy import (
+    select,
+    table,
+    column,
+    insert,
+    inspect,
+)
+
+import sqlalchemy
```

Comment:
I'm slightly confused as to why you import a bunch of specific functions you'll use from a module, and then import the whole module anyway. No big deal, just curious why you do it that way?